features/twitch-integration/index.md

```diff
@@ -0,0 +1,124 @@
+# Twitch Drops Integration
+
+## Twitch Setup
+This guide is written assuming you've never set up anything related to Twitch.
+If you have, feel free to skip ahead to relevant sections.
+
+> As you go through these steps, record the following 4 important pieces of information, as you'll need them to configure Catena:
+> - The Client Id of your Twitch Application (Step 2)
+> - The Client Secret of your Twitch Application (Step 2)
+> - Your Organization Id (Step 1)
+> - Your Game's Id (Step 1)
+
+1. [Create an Organization](https://dev.twitch.tv/docs/companies/)
+   - To create an organization, you are expected to have a game that's already known to Twitch. If you haven't done that, see [Adding a Game or Game Cover Art to Twitch](https://help.twitch.tv/s/article/adding-a-game-and-box-art-to-the-directory?language=en_US).
+   - Twitch notes that "Processing typically takes one business week," so you'll want to get this step done a short bit before you want to begin testing, otherwise you may find yourself waiting around.
+2. [Create a Twitch Application](https://dev.twitch.tv/docs/drops/#prerequisites-and-setup)
+   - For the OAuth Redirect URL, fill it in with the root URL that Catena is running on, plus the following: `/api/v1/authentication/PROVIDER_TWITCH_WEB/callback`
+   - The result should look similar to `https://catena.reallycoolgame.example.com:5000/api/v1/authentication/PROVIDER_TWITCH_WEB/callback`
+3. [Create a Reward](https://dev.twitch.tv/docs/drops/#create-a-reward)
+   1. If you have not yet created an item in Catena that you want to assign as the Twitch Drop, you should do so now.
+      - _Dev Q: Do we have a page for this?_
+      - TBD: Dashboard doc for this?
```

Comment:
I think the closest we have is within the entitlements features (https://docs.catenatools.com/features/entitlements) but this doesn't walk through the dashboard